cmd/utils/flags.go

```diff
@@ -783,6 +807,15 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config) {
 	setBootstrapNodes(ctx, cfg)
 	setBootstrapNodesV5(ctx, cfg)
 
+	if ctx.GlobalIsSet(MaxDialFlag.Name) {
+		cfg.MaxDial = ctx.GlobalInt(MaxDialFlag.Name)
+	}
+	if ctx.GlobalIsSet(MaxAcceptFlag.Name) {
+		cfg.MaxAccept = ctx.GlobalInt(MaxAcceptFlag.Name)
+	}
+	if ctx.GlobalIsSet(NoMaxPeersFlag.Name) {
+		cfg.NoMaxPeers = true
```

Comment:
Is there a config option for the number of MaxPeers? How does `NoMaxPeers` interact with this flag? Do we need to set the number of max peers?

Reply:
Nvm - I saw how it's used in the code below. Can you just add a note in the config description that it overrides the `MaxPeers` option?

Reply:
k